tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/pda.tsx

```diff
@@ -13,3 +13,10 @@ export const pda_style: Feature<string> = {
   description: 'The style of your equipped PDA. Changes font.',
   component: FeatureDropdownInput,
 };
+
+export const pda_ringtone: Feature<string> = {
+  name: 'PDA Ringtone',
+  description:
+    'Want your PDA to say something other than "beep"? Accepts the first 20 characters.',
```

Comment:
What does "Accepts the first 20 characters." mean? Is this something made redundant with proper max length filtering?

Reply:
```suggestion
    "The ringtone you'll hear when someone sends you a PDA message.",
```

Reply:
True, I copy-pasted this from the old component we had downstream, which hadn't implemented the character limit properly like I did, I'll fix that yeah.